### DISCUSSION TITLE
Editor waveform shows mono audio tracks differently

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20911] [Q&A] [answered]
Is this intended? Generally, I've noticed that audios with only 1 channel are still shown in pairs/mirrored (not sure exactly how to explain it), but here in the editor that doesn't seem to be the case:

| Mono | Stereo |
|:---:|:---:|
| ![mono](https://user-images.githubusercontent.com/43073074/197611358-b65960a6-6a0f-46fc-9c27-e9f3b31b2369.png) | ![stereo](https://user-images.githubusercontent.com/43073074/197611399-beaa563d-c784-498d-8daa-87479003ae1d.png) |

Here are those tracks in Audacity for reference:
![image](https://user-images.githubusercontent.com/43073074/197611781-1300a2ca-6a1c-4dbe-83a9-148c93fd367c.png)

**Comment by smoogipoo** [comment]
That was intended at the time, but it probably breaks expectations.

**Reply by peppy** [reply]
I'll take a look into this, seems very incorrect..

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu-framework/pull/5484
